drivers/spi/spi_max32.c

```diff
@@ -685,12 +897,13 @@ static int api_release(const struct device *dev, const struct spi_config *config
 {
 	struct max32_spi_data *data = dev->data;
 
+#ifndef CONFIG_SPI_RTIO
 	if (!spi_context_configured(&data->ctx, config)) {
 		return -EINVAL;
 	}
-
+#endif
 	spi_context_unlock_unconditionally(&data->ctx);
-
+	printf("%s", (char *)&data->ctx);
```

Comment:
Use LOG instead of printf

Reply:
This was a debug leftover, will remove it